tests/test_core.py

```diff
@@ -2849,19 +2897,23 @@ def test_nestedstructs(self):
 
         // Part 2 - the char[] should be compressed, BUT have a padding space at the end so the next
         // one is aligned properly. Also handle char; char; etc. properly.
-        B *b = NULL;
-        printf("*%ld,%ld,%ld,%ld,%ld,%ld,%ld,%ld,%zu*\\n", long(b), long(&(b->buffer)), long(&(b->buffer[0])), long(&(b->buffer[1])), long(&(b->buffer[2])),
-                                                  long(&(b->last)), long(&(b->laster)), long(&(b->laster2)), sizeof(B));
+        B b;
+        printf("*%ld,%ld,%ld,%ld,%ld,%ld,%ld,%zu*\n", long(&b.buffer) - long(&b),
+                                                      long(&b.buffer[0]) - long(&b),
+                                                      long(&b.buffer[1]) - long(&b),
+                                                      long(&b.buffer[2]) - long(&b),
+                                                      long(&b.last) - long(&b),
+                                                      long(&b.laster) - long(&b),
+                                                      long(&b.laster2) - long(&b),
+                                                      sizeof(B));
 
         // Part 3 - bitfields, and small structures
-        Bits *b2 = NULL;
-        printf("*%zu*\\n", sizeof(Bits));
-
+        printf("*%zu*\n", sizeof(Bits));
         return 0;
       }
       '''
     # Bloated memory; same layout as C/C++
-    self.do_run(src, '*16,0,4,8,8,12|20,0,4,4,8,12,12,16|24,0,20,0,4,4,8,12,12,16*\n*0,0,0,1,2,64,68,69,72*\n*2*')
+    self.do_run(src, '*16,0,4,8,8,12|20,0,4,4,8,12,12,16|24,0,20,0,4,4,8,12,12,16*\n*0,0,1,2,64,68,69,72*\n*2*')
```

Comment:
Why did a result change here?

Reply:
I removed a single print statement which was to print the pointer itself.. which was previously a null pointer so the address of zero.

Reply:
See above where `B *b = NULL;` and we use to print `long(b)` .. which is clearly always zero.

Reply:
Ah, I see now, thanks.